app/src/main/res/layout/fragment_habit_view.xml

```diff
@@ -63,4 +64,104 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/ratingHabitTextBox" />
 
+    <CheckBox
+        android:id="@+id/monday_button"
+        style="@style/day_button"
+        android:layout_width="45dp"
+        android:layout_height="45dp"
+        android:layout_marginTop="32dp"
+        android:clickable="false"
+        android:elevation="10dp"
+        android:text="Mo"
+        app:layout_constraintStart_toStartOf="@+id/descriptionHabitTextBox"
+        app:layout_constraintTop_toTopOf="parent" />
```

Comment:
These boxes are not clickable, and are just for viewing. Of note: the padding of each button is based off the Monday button, so the Monday button does not have padding here. The other buttons have differing padding though.